### ISSUE TITLE
Contact form fields do not have autocomplete hints

### ISSUE THREAD

**Issue by tremby** [tremby/markwiens.ca-ubertor#66]
On the contact page is a contact form. Its fields should have autocomplete hints so users can fill the form more quickly. These hints are absent.

![2023-08-23_22-10](https://github.com/tremby/markwiens.ca-ubertor/assets/199635/8f868064-9e4f-4bae-988c-a092b3bbab5e)
